### PR TITLE
Initial E2e fixes for post components

### PR DESCRIPTION
**Pull request by nevyangelova**
#### Summary
This branch will serve to fix all e2e tests related to the unification of the post components

#### Related Pull Requests
https://github.com/mattermost/mattermost-webapp/pull/11127/files#diff-23e10b53683ed875cd43c3096ab2a70393d66e3d4f95bc7e09b19edcea104e0d

#### Release Note
```release-note
Updated e2e tests related to the unification of post components
```

### CODE REVIEW

File: components/new_post/post_component.tsx [hunk 1]
```diff
@@ -382,7 +383,7 @@ const PostComponent = (props: Props): JSX.Element => {
             <PostAriaLabelDiv
                 ref={postRef}
                 role='listitem'
-                id={'rhsPost_' + post.id}
+                id={props.location === 'RHS_ROOT' ? `rhsPost_${post.id}` : 'postView'}
```
Comment (m1lt0n) [inline components/new_post/post_component.tsx:386]: Seeing actual messages in community, perhaps ```id={props.location === 'RHS_ROOT' ? `rhsPost_${post.id}` : `post_${post.id}`}``` along with adding a `data-testid="postView"` (currently this is in place) would fix several of the e2e test errors I think

**Review by M-ZubairAhmed — APPROVED** [review]
Haven't checked the e2es the code looks good

**Comment by nevyangelova** [comment]
merging with the parent PR to pass another QA review

**Pull request merged**